docs/how-to/contribute.md

```diff
@@ -0,0 +1,68 @@
+# How to contribute
+
+## Overview
+
+This document explains the processes and practices recommended for contributing enhancements to the Pollen operator.
+
+- Generally, before developing enhancements to this charm, you should consider [opening an issue
+  ](https://github.com/canonical/pollen-operator/issues) explaining your use case.
+- If you would like to chat with us about your use-cases or proposed implementation, you can reach
+  us at [Canonical Mattermost public channel](https://chat.charmhub.io/charmhub/channels/charm-dev)
+  or [Discourse](https://discourse.charmhub.io/).
+- Familiarising yourself with the [Charmed Operator Framework](https://juju.is/docs/sdk) library
+  will help you a lot when working on new features or bug fixes.
+- All enhancements require review before being merged. Code review typically examines
+  - code quality
+  - test coverage
+  - user experience for Juju operators of this charm.
+- Please help us out in ensuring easy to review branches by rebasing your pull request branch onto the `main` branch. This also avoids merge commits and creates a linear Git commit history.
+- Please generate src documentation for every commit. See the section below for more details.
+
+## Developing
+
+The code for this charm can be downloaded as follows:
+
+```
+git clone https://github.com/canonical/pollen-operator
+```
+
+You can use the environments created by `tox` for development:
+
+```shell
+tox --notest -e unit
+source .tox/unit/bin/activate
+```
+
+### Testing
```

Comment:
I think we could also add `src-docs` to CONTRIBUTING.md as well, [reference in Jenkins-k8s-operator](https://github.com/canonical/jenkins-k8s-operator/blob/main/CONTRIBUTING.md#generating-src-docs-for-every-commit)

Reply:
Pollen does not have src-docs as part of its tox envs, I'll add that as well.